rs/canister_client/src/http_client.rs

```diff
@@ -148,7 +148,10 @@ impl rustls::client::danger::ServerCertVerifier for DangerAcceptInvalidCerts {
     }
 
     fn supported_verify_schemes(&self) -> Vec<rustls::SignatureScheme> {
-        vec![rustls::SignatureScheme::ECDSA_NISTP256_SHA256]
+        vec![
+            rustls::SignatureScheme::ECDSA_NISTP256_SHA256,
+            rustls::SignatureScheme::RSA_PSS_SHA256,
+        ]
```

Comment:
```suggestion
        rustls::crypto::ring::default_provider().signature_verification_algorithms.supported_schemes())
```

I think it should be this so we support any scheme our crypto provider supports. But let me check again.

Reply:
I agree! I modified the code